tokio-macros/src/select.rs

```diff
@@ -73,27 +73,27 @@ fn clean_pattern(pat: &mut syn::Pat) {
             }
         }
         syn::Pat::Or(or) => {
-            for case in or.cases.iter_mut() {
```

Comment:
@Darksonn what do you think about these particular cases of `.iter_mut()` vs. `&mut`?

Reply:
I guess these are fine.